src/solvers/refinement/string_constraint_generator_main.cpp

```diff
@@ -324,16 +324,43 @@ void string_constraint_generatort::add_default_axioms(
   if(max_string_length!=std::numeric_limits<size_t>::max())
     axioms.push_back(s.axiom_for_length_le(max_string_length));
 
-  if(force_printable_characters)
-  {
-    symbol_exprt qvar=fresh_univ_index("printable", s.length().type());
-    exprt chr=s[qvar];
-    and_exprt printable(
-      binary_relation_exprt(chr, ID_ge, from_integer(' ', chr.type())),
-      binary_relation_exprt(chr, ID_le, from_integer('~', chr.type())));
-    string_constraintt sc(qvar, s.length(), printable);
-    axioms.push_back(sc);
-  }
+  const exprt index_zero = from_integer(0, s.length().type());
+  if(m_force_printable_characters)
```

Comment:
No `m_` prefixes please.

Reply:
this field will be removed by the last commit anyway

Reply:
actually there may be a rebase issue there, I will check the commit still compiles

Reply:
done, the history should be clean now